roottest/root/io/compression/CMakeLists.txt

```diff
@@ -1,7 +1,30 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOTTEST_GENERATE_DICTIONARY(libEventDict
```

Comment:
Now that the name are in the 'global' CMake namespace, do we need to disambiguate them?

Reply:
Yes, I just recognize that library name also create **global** target of same name.
I propose to add `LIBNAME` parameter to `ROOTTEST_GENERATE_DICTIONARY` - 
and then use more unique target names